algorithms/sort/heap_sort.py

```diff
@@ -1,17 +1,21 @@
-def max_heap_sort(arr):
+def max_heap_sort(arr, simulation = False):
```

Comment:
please remove the space around ` = ` in arguments.
```python
def max_heap_sort(arr, simulation=False):
```
Same changes in all function definitions.

Reply:
I changed it,
Thank you for your comment! :)